githubscript/tests/test_release_config.py

```diff
@@ -43,6 +43,131 @@ def _dummy_get_artifacts(*args, **kwargs):
     }
 
 
+def test_release_config_override(monkeypatch):
+    def _dummy_get_artifacts(*args, **kwargs):
+        return [{"content_type": "application/vnd.android.package-archive", "local_path": "/dummy/path/to.apk", "name": "somme_pretty_name.apk", "size": 9000}]
+
+    monkeypatch.setattr(release_config, "_get_artifacts", _dummy_get_artifacts)
+
+    config = {}
+
+    product_config = {
+        "github_owner": "config_owner",
+        "github_repo_name": "config_repo",
+        "allow_github_repo_override": True,
+        "github_token": "some_secret_token",
+        "contact_github": True,
+    }
+
+    task_payload = {
+        "githubOwner": "task_owner",
+        "githubRepoName": "task_repo",
+        "gitRevision": "somecommithash",
+        "gitTag": "v1.0.0",
+        "isPrerelease": False,
+        "releaseName": "SomeProduct v1.0.0",
+    }
+
+    assert release_config.get_release_config(product_config, task_payload, config) == {
+        "artifacts": [
+            {"content_type": "application/vnd.android.package-archive", "local_path": "/dummy/path/to.apk", "name": "somme_pretty_name.apk", "size": 9000}
+        ],
+        "contact_github": True,
+        "git_revision": "somecommithash",
+        "git_tag": "v1.0.0",
+        "github_owner": "task_owner",
+        "github_repo_name": "task_repo",
+        "github_token": "some_secret_token",
+        "is_prerelease": False,
+        "release_name": "SomeProduct v1.0.0",
+    }
+
+
+def test_release_config_override_false(monkeypatch):
```

Comment:
We might be able to avoid 3 different tests by using `@pytest.mark.parametrize`, but I don't think that needs to block.